_format/1.1/index.md

```diff
@@ -585,6 +596,96 @@ that includes a [profile alias][profile aliases]:
+    a direct or indirect relationship to be established with the link's context.
+
+> Note: These link relation types correspond to operations described by the
+JSON:API specification and it is expected that implementations will use them to
+reference resources that will process requests according to [the requirements for
+creating, updating or deleting resources](https://jsonapi.org/format/1.1/#crud). However, this is not a strict
+limitation on their use.
+
+In the example below, two links are established. A `self` link indicating the
+URL of the request that generated the current collection of articles and an `add`
+link indicating the that the client can add an article to this collection.
+
+```json
+"links": {
+  "self": {
+    "href": "http://example.com/articles",
+    "rel": ["self", "add"]
+  }
+}
+```
+
+In the next example, a `comment` link is a member of a `comments` [relationship
+object][relationships] and that relationship object is a member of an `article`'s relationships
+object. In this case, the `rel` member establishes two links indicating that
+the client can create a new comment at the target URL and that requests to the
+target `href` can create a relationship to the article's `comments` field.
+
+```json
+"links": {
+  "self": "http://example.com/articles/1/relationships/comments",
+  "related": "http://example.com/articles/1/comments",
+  "comment": {
+    "href": "http://example.com/articles/1/comments",
+    "rel": ["add", "relate"]
+  }
+}
+```
+
+In this example, a `self` link is a member of an `article`'s links object. In
+this case, the `rel` member established two links indicating that the client can
+edit, but not delete, the context article (perhaps another request with different
+authentication credentials would receive an additional link with a `remove` link
+relation type).
+
+```json
+"links": {
+  "self": {
+    "href": "http://example.com/articles/1",
+    "rel": ["self", "update"]
+  }
+}
+```
+
+#### <a href="#document-links-link-parameter-object" id="document-links-link-parameter-object" class="headerlink"></a> Link parameter objects
+
+"Link parameter objects" appear in link objects to represent a link's target
+attributes.
+
+A link parameter object **MAY** contain any of the following members: `hreflang`,
+`media`, `title`, and `type`. The values of these members **MUST** conform to their
+meanings as defined by [RFC8288 Section 3.4.1](https://tools.ietf.org/html/rfc8288#section-3.4.1).
+
+A link parameter object **MAY** contain other members as target attributes. Any
+additional member names **MUST** be valid target attribute names as defined by
+[RFC8288 Section 2.2](https://tools.ietf.org/html/rfc8288#section-2.2).
+
+Link parameter object members with multiple values **MUST** be represented with an
+array (i.e. a target attribute with multiple values should be an array of
+values, not a whitespace-separated string).
+
+A link parameter object **SHOULD NOT** contain a `rev` member.
```

Comment:
This seems oddly specific compared to everything else. But @dgeb didn't comment on this, so probably this is my being insufficiently informed to understand this without additional context.

Reply:
Removed. I've amended the rules for member names such that they must conform to the rules for member names in any other place in the document. Therefore, `rev` is not a valid member name by definition.

I've left in the guidance on how to represent reversed links, but taken out this over-specific normative statement.